app/static/js/index.js

```diff
@@ -347,3 +347,24 @@ window.onscroll = function(e) {
         pagesFinished = true;
     }
 }
+
+// Scroll to top button
+var scrollTopBtn = document.getElementById('scrollToTopBtn')
+var rootElement = document.documentElement
+
+function showButton() {
+if ((rootElement.scrollTop / (rootElement.scrollHeight - rootElement.clientHeight)) > 0.15 ) {
```

Comment:
Some substitutions that should be a little cleaner and remove the need to use `document.documentElement`:

`rootElement.scrollTop` -> `window.scrollY`

`rootElement.clientHeight` -> `document.innerHeight`

I think you can do this check more clearly by checking if, say, `window.scrollY > 2 * document.innerHeight`. In words, check if the user has scrolled more than 2 screen heights downward

Reply:
Also make sure this is indented properly